modules/SCsub

```diff
@@ -19,5 +19,6 @@ for x in env.module_list:
     SConscript(x + "/SCsub")
 
 lib = env_modules.Library("modules", env.modules_sources)
+env.NoCache(lib)
```

Comment:
Isn't there a way to define that no libs should have cache, instead of hardcoding it everywhere?

Or at least it should be moved to a helper method, like `methods.add_library(env, name, sources)`.

Reply:
Well we can improve this later on.